### DISCUSSION TITLE
How to configure numbers on X and Y axis in X/Y plugin?

### DISCUSSION THREAD

**Discussion by githubkgp** [perspective-dev/perspective#2108] [Q&A]
Numbers to be populated in X and Y axis are too small ( 0.0012345, 0.006789... ). But in the X/Y line graph all points in X and Y axis are just shown as 0.001. I would like  to define the precision of numbers and by how much each data points on axis must be differentiated (Example: 0.0012**345**, 0.0012**350**, 0.0012**400**....). Also in the tool tip the numbers are just shown with 2 decimal points. I would also like to represent them with entire precision
I would not like to format the numbers and show in the graph. I would like to represent the numbers with entire precision.

**Comment by Himasnhu-AT** [comment]
That sound nice and helpful. I'm new to open-source contribution, can I work on it with you, so that I can understand how open source works.
